### DISCUSSION TITLE
REI Not Showing Recipes!

### DISCUSSION THREAD

**Discussion by NoShiSherlock** [shedaniel/RoughlyEnoughItems#1228] [Q & A]
Hello!

My friend is running a server for a couple friends and myself, and we are struggling with REI. It seems that it isn't showing any recipes. We have tried to make sure every bit of configurable option is enabled/disabled that might be causing the issue. It works in single player, but not multiplayer? 

What to note:
The server is hosted on Pebble host, but never had the issue on other custom mod packs. 
With that being said, it is a CUSTOM mod pack we are running.
It works perfectly fine in single player.
We have tried to add/remove mods 1 by 1 to see if that did anything. 

SCREENSHOT: https://user-images.githubusercontent.com/118637049/202873029-31bb0406-2e35-4a6e-ab5b-595e98f6571c.jpg

**Comment by shedaniel** [comment]
That's not even REI, you are not using REI in your image

**Reply by NoShiSherlock** [reply]
I'm aware. But the point is that the same issue is happening with all Item Display mods.

**Reply by Nerdtality** [reply]
https://modrinth.com/mod/show-me-your-skin causes this same issue.
